app/src/main/java/com/jerboa/ui/components/community/list/CommunityListActivity.kt

```diff
@@ -55,8 +60,20 @@ fun CommunityListActivity(
 
     val scope = rememberCoroutineScope()
 
+    val baseModifier = if (padding == null) {
+        Modifier
+    } else {
+        // https://issuetracker.google.com/issues/249727298
```

Comment:
When using edge to edge, it also applies the navigationbar padding above the IME